makefile.shade

```diff
@@ -125,13 +128,44 @@ var buildTarget = "compile"
 
         Parallel.ForEach(repositories, repo =>
         {
+            var useBuildBranch = true;
             if (Directory.Exists(repo))
             {
-                GitCommand("", string.Format("pull --ff-only --quiet --depth 1 git@github.com:aspnet/{1}.git {0}:{0}", buildBranch, repo));
+                if(UseGateBranch)
+                {
+                    try
+                    {
+                        GitCommand("", string.Format("pull --f-only --quite --depth 1 git@github.com:aspnet/{1}.git {0}:{0}", gateBranch, repo));
+                        useBuildBranch = false;
```

Comment:
We should leave this on. It could be likely that some repos don't have the gate-branch but others do.